### PR TITLE
chore: Adding Mnist & ObjectDetection models

### PR DESCRIPTION
**Pull request by aravindhank11**

### CODE REVIEW

File: eva/executor/create_udf_executor.py [hunk 1]
```diff
@@ -70,8 +70,8 @@ def exec(self):
         )
 
         # Profile the UDF
-        sample = catalog_manager.get_udf_profiler_sample_by_type(self.node.udf_type)
-        profiler = Profiler(impl_path, self.node.name, sample.sample_path, sample.validation_path)
+        #sample = catalog_manager.get_udf_profiler_sample_by_type(self.node.udf_type)
+        profiler = Profiler(impl_path, self.node.name, "", "")#, sample.sample_path, sample.validation_path)
```
Comment (aravindhank11) [inline eva/executor/create_udf_executor.py:74]: @NetraGhaisas please fix this in your next PR

File: eva/udfs/object_detector.py [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+class ObjectDetector(PytorchAbstractClassifierUDF):
+    """
+    Arguments:
+        threshold (float): Threshold for classifier confidence score
+    """
+
+    def __init__(self, model_name):
+        module = importlib.import_module("torchvision.models.detection")
+        self._torch_model = getattr(module, model_name)
+        super().__init__()
```
Comment (aravindhank11) [inline eva/udfs/object_detector.py:55]: This code is essentially a copy paste of fasterrcnn example except for this minor modification. 

This is all we need to bring in any model supported by `torchvision.models.detection`

File: eva/udfs/object_detector.py [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+    def __init__(self):
+        super().__init__("ssdlite320_mobilenet_v3_large")
+
+class SsdVggObjectDetector(ObjectDetector):
+    def __init__(self):
+        super().__init__("ssd300_vgg16")
+
+class KeypointObjectDetector(ObjectDetector):
+    def __init__(self):
+        super().__init__("keypointrcnn_resnet50_fpn")
```
Comment (aravindhank11) [inline eva/udfs/object_detector.py:224]: Create a child class of `ObjectDetector` and pass the model of interest as a string.

File: eva/udfs/object_detector.py [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+if __name__ == "__main__":
```
Comment (aravindhank11) [inline eva/udfs/object_detector.py:226]: Driver code for testing

**Review by sanjeevumeshrao — APPROVED** [review]

**Pull request merged**